GameServerLib/Logic/GameObjects/Spell.cs

```diff
@@ -126,22 +126,6 @@ public Spell(Champion owner, string spellName, byte slot)
 
             JObject data;
 
-            if (slot > 3)
```

Comment:
The code that follows that block does the same and more. Returning at that point means that the cast time is never loaded from the files